src/luarocks/fetch/cvs.lua

```diff
@@ -20,9 +20,14 @@ function cvs.get_sources(rockspec, extract, dest_dir)
    assert(type(rockspec) == "table")
    assert(type(dest_dir) == "string" or not dest_dir)
 
+   local cvs_cmd = rockspec.variables.CVS
+   if not fs.execute_quiet(cvs_cmd, "--version") then
+      return nil, "'"..cvs_cmd.."' program not found. Is CVS installed? You may want to edit variables.CVS"
```

Comment:
wouldn't it be clearer to write; "You may want to edit the 'variables.CVS' value in file 'config.lua' "